README.md

```diff
@@ -38,6 +38,9 @@ $ geth --help
     --builder                      (default: false)
           Enable the builder
 
+    --builder.algotype value       (default: "mev-geth")
```

Comment:
equivalent miner configs should be removed

Reply:
based on your offline feedback, i've retained original `miner.algotype` flag for backwards compatibility, please feel free to un-resolve if there are still concerns